src/miner_hbbft_sidecar.erl

```diff
@@ -85,6 +86,13 @@ new_round(Buf, BinTxns) ->
 prefilter_round(Buf, Txns) ->
     gen_server:call(?SERVER, {prefilter_round, Buf, Txns}, infinity).
 
+handle_txn(submit = RequestType, Txn) ->
+    lager:debug("handling txn with request type ~p", [RequestType]),
+    gen_server:call(?SERVER, {submit, Txn}, infinity);
+handle_txn(update = RequestType, Txn) ->
+    lager:debug("handling txn with request type ~p", [RequestType]),
```

Comment:
these two debug log statements look like they're just extra noise in context of the debug statements we're also logging in their accompanying `handle_call/3` functions; would propose consolidating those into a single statement in the handler.

Reply:
done